rules/manager.go

```diff
@@ -212,8 +212,12 @@ func (g *Group) copyState(from *Group) {
 			if !ok {
 				continue
 			}
-			if far.Name() == ar.Name() {
-				ar.active = far.active
+			// TODO(fabxc): forbid same alert definitions that are not unique by
```

Comment:
It's fine to have alerts that share these, it tends to be a lot cleaner than doing it with `or`.